diff --git a/node_modules/@reactive-dot/core/.bun-tag-a982c29b69b00b08 b/.bun-tag-a982c29b69b00b08
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/build/providers/light-client/wellknown-chains.js b/build/providers/light-client/wellknown-chains.js
index a0354f4c010becb9f56fe5588658245526c10cfe..3824febac842f3ed58052866d666d41e8c640b8f 100644
--- a/build/providers/light-client/wellknown-chains.js
+++ b/build/providers/light-client/wellknown-chains.js
@@ -19,7 +19,10 @@ export const wellknownChains = {
     ],
     paseo: [
         () => import("polkadot-api/chains/paseo"),
-        { paseo_asset_hub: () => import("polkadot-api/chains/paseo_asset_hub") },
+        { 
+            paseo_asset_hub: () => import("polkadot-api/chains/paseo_asset_hub"),
+            paseo_people: () => import("polkadot-api/chains/paseo_people"),
+        },
     ],
     westend: [
         () => import("polkadot-api/chains/westend2"),
diff --git a/src/providers/light-client/wellknown-chains.ts b/src/providers/light-client/wellknown-chains.ts
index 090324e82a114a1ecd1ac20ec63aac7a9c91f075..4a740e06eca0f942f3df6ac1c35b89021d6f387d 100644
--- a/src/providers/light-client/wellknown-chains.ts
+++ b/src/providers/light-client/wellknown-chains.ts
@@ -22,7 +22,10 @@ export const wellknownChains = {
   ],
   paseo: [
     () => import("polkadot-api/chains/paseo"),
-    { paseo_asset_hub: () => import("polkadot-api/chains/paseo_asset_hub") },
+    { 
+      paseo_asset_hub: () => import("polkadot-api/chains/paseo_asset_hub"),
+      paseo_people: () => import("polkadot-api/chains/paseo_people") ,
+    },
   ],
   westend: [
     () => import("polkadot-api/chains/westend2"),
