diff --git a/lib/data-patch.js b/lib/data-patch.js
index cd49cbec3e7de20dea750e95e007dbdb1488ac96..854ecd25d5ce2940468de6ad8162cb26085fbed5 100644
--- a/lib/data-patch.js
+++ b/lib/data-patch.js
@@ -1,6 +1,3 @@
-import { createRequire } from 'module';
-
-const require = createRequire(import.meta.url);
-const patch = require('../data/patch.json');
+import * as patch from "../data/patch.json"
 
 export default patch;
diff --git a/lib/version.js b/lib/version.js
index dde3a83f43883ecbad6ca0a23afb58c12235148a..ab622ddeca28b3aa240ce64905e46015c208710b 100644
--- a/lib/version.js
+++ b/lib/version.js
@@ -1,5 +1 @@
-import { createRequire } from 'module';
-
-const require = createRequire(import.meta.url);
-
-export const { version } = require('../package.json');
+export { version } from "../package.json";
